tests/DCPS/Compiler/is_topic_type/run_test.pl

```diff
@@ -76,19 +79,47 @@ sub subtest {
       $status = 1;
     }
   }
-  # Make sure we found everything in %expected
-  while (my ($k, $v) = each %expected) {
-    if ($v->{not_found}) {
-      print STDERR "ERROR in $mode: $k was not found in ITL\n";
+  return $status;
+}
+
+sub subtest2 {
```

Comment:
Copying the function is overkill. Have it take the filename as an argument and combine the hashes like I suggest above.